addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,351 @@
+        <state readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="rainrate-channel">
+        <item-type>Number</item-type>
+        <label>Rain Rate Channel</label>
+        <description>Rain fall rate in millimeters per hour</description>
+        <state pattern="%d mm/h" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="raintotal-channel">
+        <item-type>Number</item-type>
+        <label>Rain Total Channel</label>
+        <description>Total rain in millimeters</description>
+        <state pattern="%d mm" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="shutter-channel">
+        <item-type>Rollershutter</item-type>
+        <label>Shutter Channel</label>
+        <description></description>
+    </channel-type>
+
+    <channel-type id="instantpower-channel">
+        <item-type>Number</item-type>
+        <label>Instant Power Channel</label>
+        <description>Instant power consumption in Watts</description>
+        <state pattern="%d W" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="totalusage-channel">
+        <item-type>Number</item-type>
+        <label>Total Usage Channel</label>
+        <description>Used energy in Watt hours</description>
+        <state pattern="%d Wh" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="instantamp-channel">
+        <item-type>Number</item-type>
+        <label>Instant Amp Channel</label>
+        <description>Instant current in Amperes</description>
+        <state pattern="%d A" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="totalamphours-channel">
+        <item-type>Number</item-type>
+        <label>Total Ampere-hours Channel</label>
+        <description>Used "energy" in ampere-hours</description>
+        <state pattern="%d Ah" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="temperature-channel">
+        <item-type>Number</item-type>
+        <label>Temperature Channel</label>
+        <description>Current temperature in degree Celsius</description>
+         <state pattern="%d °C" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="humidity-channel">
+        <item-type>Number</item-type>
+        <label>Humidity Channel</label>
+        <description>Relative humidity level in percentages</description>
+        <state min="0" max="100" step="1" pattern="%d %" readOnly="true"></state>
+    </channel-type>
+
+	<channel-type id="humiditystatus-channel">
+		<item-type>String</item-type>
+		<label>Humidity Status Channel</label>
+		<description>Current humidity status</description>
+		<state readOnly="true">
+			<options>
+				<option value="NORMAL">Normal humidity</option>
+				<option value="COMFORT">Comfort humidity</option>
+				<option value="DRY">Dry humidity</option>
+				<option value="WET">Wet humidity</option>
+			</options>
+		</state>
+	</channel-type>
+
+    <channel-type id="signallevel-channel">
```

Comment:
Check whether you could offer the system channel type for signal-strength, see https://github.com/eclipse/smarthome/blob/master/docs/sources/architecture/thing-definition.md#channels.
This would allow UIs to nicely display this information